### DISCUSSION TITLE
address list

### DISCUSSION THREAD

**Discussion by RightSexyOrc** [Chia-Network/chia-blockchain#15226] [Ideas]
suggest adding an address list display, e.g. somewhat like bitcoin Electrum wallet

Serious ux issue in new versions of chia ref wallet, fyi  , now that it is creating new addresses all the time. Namesholders are giving us feedback that they are really confused by the changing address. I note that in Bitcoin, Electrum wallet lets you display list of addresses easily and see which have balance (coins) in them . I don't think we have that capability in chia ref wallet, and it seems like that's needed urgently. we've had a bump in new support tickets related to this, and it is hard to get people to a comfort level about which are really the addresses in their wallets. we're having to try to explain it multiple times, and they are using chia less than if it were clearer and are concerned about their investments in NFTs. It'd be nice to easily get a list of currently used addresses (including ones that have NFT's, not just xch and tokens), preferably in the gui.

**Comment by forReason** [comment]
suggesting merge with #15360
